src/encoder.rs

```diff
@@ -3473,8 +3546,16 @@ fn encode_tile<'a, T: Pixel>(
       cw.bc.cdef_coded = false;
       cw.bc.code_deltas = fi.delta_q_present;
 
+      let is_straddle_sbx =
+        tile_bo.0.x + BlockSize::BLOCK_64X64.width_mi() > ts.mi_width;
+      let is_straddle_sby =
+        tile_bo.0.y + BlockSize::BLOCK_64X64.height_mi() > ts.mi_height;
+
       // Encode SuperBlock
-      if fi.config.speed_settings.encode_bottomup {
+      if fi.config.speed_settings.encode_bottomup
+        || is_straddle_sbx
```

Comment:
What this change does is: If SB straddle on frame boundary, use bottom-up partition search because topdown cannot use open-partition due to its limited search depth of one level.
However, currently I observe this change CAUSE DESYNC, i.e. mismatch between the decoded video from encoder and decoder, so I think I need to fix this before landing.